recipe/meta.yaml

```diff
@@ -116,83 +116,119 @@ build:
+        - binutils_impl_{{ target_platform }} >=2.34  # [linux]
+      host:
+        - bzip2  # [not win]
+        - sqlite
+        - xz  # [not win]
+        - zlib  # [not win]
+        - openssl
+        - readline  # [not win]
+        - tk  # [not win]
+        - ncurses  # [linux]
+        - libffi  # [not win]
+        - ld_impl_{{ target_platform }} >=2.34  # [linux]
+      run:
+        - ld_impl_{{ target_platform }} >=2.34  # [linux]
+      run_constrained:
+        - python_abi 3.8.* *_cp38
+    test:
+      requires:
+        - ripgrep
+        - cmake
+        - ninja
+        - cython
+        - setuptools
+        - {{ compiler('c') }}
+        # Tried to use enable_language(C) to avoid needing this. It does not work.
+        - {{ compiler('cxx') }}
+      files:
+        - tests/distutils/*
+        - tests/distutils.cext/*
+        - tests/cmake/*
+        - tests/cython/*
+        - tests/prefix-replacement/*
+      commands:
+        - echo on  # [win]
+        - set  # [win]
+        - python -V
+        - python3 -V            # [not win]
+        - 2to3 -h
+        - pydoc -h
+        - python3-config --help  # [not win]
+        - set "PIP_NO_BUILD_ISOLATION=False"  # [win]
+        - set "PIP_NO_DEPENDENCIES=True"  # [win]
+        - set "PIP_IGNORE_INSTALLED=True"  # [win]
+        - set "PIP_NO_INDEX=True"  # [win]
+        - set "PIP_CACHE_DIR=%CONDA_PREFIX%/pip_cache"  # [win]
+        - set "TEMP=%CONDA_PREFIX%/tmp"  # [win]
+        - mkdir "%TEMP%"  # [win]
+        - python -Im ensurepip --upgrade --default-pip  # [win]
+        - python -m venv test-venv
+        - python -c "import sysconfig; print(sysconfig.get_config_var('CC'))"  # [not win]
+        -  _CONDA_PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_x86_64_conda_cos6_linux_gnu python -c "import sysconfig; print(sysconfig.get_config_var('CC'))"  # [linux64]
+        # check for unreplaced @ symbols in sysconfig files
+        - for f in ${CONDA_PREFIX}/lib/python*/_sysconfig*.py; do echo "Checking $f:"; if [[ `rg @ $f` ]]; then echo "FAILED ON $f"; cat $f; exit 1; fi; done  # [linux64 or osx]
+        - test ! -f ${PREFIX}/lib/libpython3.8.a  # [unix]
+        - test ! -f ${PREFIX}/lib/libpython3.8.nolto.a  # [unix]
+        - pushd tests
+        -   pushd distutils
+        -     python setup.py install -v -v
+        -     python -c "import foobar"
+        -   popd
+        -   pushd distutils.cext
+        -     python setup.py install -v -v
+        -     python -c "import greet; greet.greet('Python user')" | rg "Hello Python"
+        -   popd
+        -   pushd prefix-replacement  # [unix]
+        -     bash build-and-test.sh  # [unix]
+        -   popd  # [unix]
+        -   pushd cmake
+        -     cmake -GNinja -DPY_VER={{ version }}
+              # --trace --debug-output --debug-trycompile .
+        -   popd
+        - popd
+        - test ! -f default.profraw   # [osx]
+  - name: libpython-static
```

Comment:
Shouldn't this just be `python-static` to follow the convention?

Reply:
That would be a bit confusing in this case as the main Python is also statically linking libpython. As it's only the lib what is included in the -static package, I would rather reflect that also in the name.

Reply:
It still seems a bit odd to me.  I am not blocking based on the name though.

Reply:
Any other opinions? I would happily sacrifice the name for a merge but I still think `libpython-static` is more intuitive regarding the contents of the new output.

Reply:
I am also happy to be outvoted here too.